docs/pages/contributing/documentation/style-guide.mdx

```diff
@@ -150,71 +309,67 @@ Some guides are intended for end users seeking to access resources in their
 cluster. For certain use cases, it may be necessary to adjust our usual voice
 for the audience of a specific guide.
 
-### Body text
-
-- Write sparser one to two-sentence groupings rather than lengthier blocks of text.
-- Use periods at the end of a line even in a list unless the ending item is a command.
-
 ### Code, commands, and configuration
 
 - `tsh`, `tctl`, and other core commands should be placed in backticks.
 
-- All ports or values should be enclosed in backticks, e.g., `443`.
+- All ports or values should be enclosed in backticks, for example, `443`.
```

Comment:
I'm not a fan of this one. I would prefer we use backticks when we really want a monospaced font (things like code snippets, configuration files, commands, etc). Otherwise we just add extra visual load without good reason.

For example, is:

Teleport listens on port `3080` by default.

really better than

Teleport listens on port 3080 by default.

??